Docker image does not run

`docker run` fails with `Error: Cannot find module '@server/env'`.

**To Reproduce**
Steps to reproduce the behavior:
1. `docker-compose build`
2. `docker run --rm -it outline:v001`
3. See error

**Expected behavior**
No `MODULE_NOT_FOUND` error.

**Screenshots**
```
yarn run v1.22.15
$ node ./build/server/index.js
internal/modules/cjs/loader.js:905
  throw err;
  ^

Error: Cannot find module '@server/env'
Require stack:
- /opt/outline/build/server/logging/logger.js
- /opt/outline/build/server/index.js
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:902:15)
    at Function.Module._load (internal/modules/cjs/loader.js:746:27)
    at Module.require (internal/modules/cjs/loader.js:974:19)
    at require (internal/modules/cjs/helpers.js:93:18)
    at Object.<anonymous> (/opt/outline/build/server/logging/logger.js:12:35)
    at Module._compile (internal/modules/cjs/loader.js:1085:14)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Module.require (internal/modules/cjs/loader.js:974:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/opt/outline/build/server/logging/logger.js',
    '/opt/outline/build/server/index.js'
  ]
}
error Command failed with exit code 1.
```

**Outline (please complete the following information):**
- Install: [self hosted]
- Version: [2db49820f9660a7f957ea3f50b1362f27e001b4f]

--- a/.dockerignore
+++ b/.dockerignore
@@ -15,4 +15,3 @@ build
 docker-compose.yml
 fakes3
 node_modules
-tsconfig.json

